### DISCUSSION TITLE
Behavior when long pressing "a", "o", "u" + "s" on the German keyboard with separate keys for ä, ö, ü + ß

### DISCUSSION THREAD

**Discussion by samarutuk** [florisboard/florisboard#2416] [Feedback]
Hi, I'm using the extended German keyboard layout with its own keys for the umlauts (ä, ö, ü) and the "ß". In the reduced layout, you press and hold the "a", "o" or "u" key for the umlauts and the "s" key for the "ß". Unfortunately, the extended German keyboard with separate keys for umlauts and "ß" still does this. However, since "ß" and the umlauts have their own keys, the automatic selection when pressing a key for a longer time should display the alternative assignment of the key shown in the top right corner, i.e. "#" for "s" (not "ß"), "<" for "u" (not "ü", which has an extra key), "{" for "o" and - very important - "@" for "a" (not "ä"). If anyone could change this, that would be great!

Regards
Sam

P.S.: What always looks a bit strange is the underscore "_", which sticks very close to the "f" due to its position. If you move it up a bit, it will certainly still be perceived as an underscore, but it would be better separated from the "f".

**Comment by patrickgold** [comment]
That can be solved with an alternative German popup mapping not mapping the äöüß popups as the main popup chars

**Reply by samarutuk** [reply]
_That would be really great!_ Selecting the desired character via pop-up often slows me down, especially when writing a lot. And based on the layout of the German keyboard with keys for ä,ö,ü,ß plus the special characters displayed at the top right, it would be the expected behavior to get the special character with a long keystroke.
